concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -681,8 +681,8 @@ Hale-1973-1798-679	House_and_Home-21	post	n	06a.21	281	POST
 Hale-1973-1798-680	House_and_Home-22	bamboo ties (strips for building)	n	06a.22		
 Hale-1973-1798-681	House_and_Home-23	scaffold	n	06a.23		
 Hale-1973-1798-682	House_and_Home-24	ladder	n	06a.24	1464	LADDER
-Hale-1973-1798-683	House_and_Home-25	veranda	n	06a.25		
-Hale-1973-1798-684	House_and_Home-26	doorway	n	06a.26		
+Hale-1973-1798-683	House_and_Home-25	veranda	n	06a.25	3269	PORCH
```

Comment:
According to  [WordNet](http://wordnetweb.princeton.edu/perl/webwn?o2=&o0=1&o8=1&o1=1&o7=&o5=&o9=&o6=&o3=&o4=&s=VERANDA&i=2&h=100#c) "veranda" is a hypernym of "porch". So would unmap.

Reply:
@mathildavz Please leave the mapping as is. We go with the existing mappings in Concepticon and we already have another _veranda_ gloss mapped to PORCH.